docs/data/charts/pie/pie.md

```diff
@@ -6,10 +6,79 @@ title: Charts - Pie
 
 <p class="description">Pie charts express portions of a whole, using arcs or angles within a circle.</p>
 
-:::warning
-This feature isn't implemented yet. It's coming.
+## Basics
 
-👍 Upvote [issue #7884](https://github.com/mui/mui-x/issues/7884) if you want to see it land faster.
+To plot a pie chart, a series must have a data property containing an array of objects.
+Those objects should contain a property `value`.
+They can also have a `label` property.
 
-Don't hesitate to leave a comment on the same issue to influence what gets built. Especially if you already have a use case for this component, or if you are facing a pain point with your current solution.
-:::
+If you plan to update/reorder those data, you should add an `id` property which is used for `key` props.
+
+{{"demo": "BasicPie.js"}}
+
+## Colors
+
+The pie colors can be customized in two ways.
+
+1. You can provide a [color palette](/x/react-charts/styling/#color-palette). Each arc of the pie will be colored according to this palette.
+2. You can provide a `color` property in `data` objects which overrides the palette.
+
+```jsx
+<PieChart
+  colors={['red', 'blue', 'green']} // Use palette
+  series={[
+    {
+      data: [
+        { value: 10, color: 'orange' }, // Use color property
+        // ...
+      ],
+    },
+  ]}
+/>
+```
+
+{{"demo": "PieColor.js"}}
+
+## Sizing
+
+Pie series shape is described by multiple properties:
+
+- `innerRadius` The radius between the center and the beginning of the arc. The default is set to 0.
+- `outerRadius` The radius between the center and the end of the arc. The default is the largest value available in the drawing area.
+- `paddingAngle` The angle (in degrees) between two arcs.
+- `cornerRadius` Similar to the CSS `border-radius`.
+- `startAngle`/`endAngle` The angle range of the pie chart. Values are given in degrees.
+- `cx`/`cy` The center of the pie charts. By default the middle of the drawing area.
+
+{{"demo": "PieShapeNoSnap.js", "hideToolbar": true, "bg": "inline", "disableAd": true}}
+
+## Labels
+
+You can display labels on the arcs.
+To do so, the series should get `arcLabel` property.
+It can either get a function that gets the object associated with the arc and returns the label.
+Or you can pass one of the following values:
+
+- `'value'` display the raw value of the arc.
+- `'formattedValue'` display the returned value of `valueFormatter` for the arc.
+- `'label'` display the `label` property of the arc if provided.
+
+To avoid displaying labels on small arcs, you can provide `arcLabelMinAngle` property.
+Arcs with angles smaller than the value (in deg) will not have labels.
+
+{{"demo": "PieArcLabel.js"}}
```

Comment:
This could be a good opportunity to showcase customization classes to color the arcLabel as white (given our palette) and use a smaller font-size.